tests/handlers-spec.ts

```diff
@@ -52,6 +52,57 @@ describe('handleChromiumCheck()', () => {
       });
     });
 
+    it('properly fetches supported versions of Electron to roll against', async () => {
+      this.mockOctokit.repos.listBranches.mockReturnValue({
```

Comment:
it's a crime that this isn't called `mocktokit`

Reply:
truly